brainf/interpreter.py

```diff
@@ -18,10 +18,22 @@ def __init__(self, table_size):
         self.size = table_size
         self.table = [0 for _ in range(table_size)]
         self.index = 0
+        self._code = None
```

Comment:
Po co self._code? bo nie widzę jakiegoś szerokiego zastosowania.

Reply:
żeby wyrzucić do metody funkcjonalność odpowiadającą za początek pętli ("["). Jak na początku `interpret()` wrzucamy przekazany kod do obiektu to możemy zrobić `_start_loop()` bez żadnego argumentu.